sentences/en/person_HassGetState.yaml

```diff
@@ -0,0 +1,44 @@
+language: en
+intents:
+  HassGetState:
+    data:
+      # https://www.home-assistant.io/integrations/person/
+      - sentences:
+          - "<where_is> <name>"
+        response: where
+        requires_context:
+          domain: person
+        slots:
+          domain: person
+
+      - sentences:
+          - "is <name> at {zone:state}"
```

Comment:
Can you make `at` optional? I think we should be able to ask "is Bob home?"

Reply:
Maybe also add a `[the]`. E.g "Is Bob at the kindergarden?"

Reply:
I could make the at optional but I would like to make the answer just use the same preposition as the question, I think this should be possible with lists but I can't see how.

Reply:
Keeping the preposition would require extending the `HassGetState` event to accept a new slot, which I'd prefer not to do.  If you want to keep the same preposition, I'd suggest instead creating separate responses for each case: `where_at`, `where_in`, etc.

Reply:
@JosephAbbey are you willing to split it into separate sentences and responses like Mike suggested or should I merge it as it is?

Reply:
Currently I am only happy with the `where is...` sentence because of the `at` problem, so I will create a separate pr with just that one